src/test-utilities/PolarisTestProvider.tsx

```diff
@@ -1,6 +1,5 @@
 import React from 'react';
-// eslint-disable-next-line shopify/strict-component-boundaries
-import {FrameContext, FrameContextType} from '../components/Frame';
+import {FrameContext, FrameContextType} from '../utilities/frame';
```

Comment:
Can we stop exposing the `FrameContextType` and instead use the `React.ContextType<typeof FrameContext>` inference instead to keep our API surface area small.